src/variants.js

```diff
@@ -55,89 +97,122 @@ export async function getVariant(name) {
 	return variants.find(variant => variant.name === name);
 }
 
-export const shortForms = /** @type {const} */ ({
-	'Red': 'r',
-	'Yellow': 'y',
-	'Green': 'g',
-	'Blue': 'b',
-	'Purple': 'p',
-	'Teal': 't',
-	'Black': 'k',
-	'Rainbow': 'm',
-	'White': 'w',
-	'Pink': 'i',
-	'Brown': 'n',
-	'Omni': 'o',
-	'Null': 'u',
-	'Prism': 'i'
-});
+export let shortForms = /** @type {string[]} */ (['r', 'y', 'g', 'b', 'p']);
+
+/**
+ * Edits shortForms to have the correct acryonyms.
+ * @param {Variant} variant
+ */
+export async function getShortForms(variant) {
+	const colors = await colours_promise;
+	const abbreviations = [];
+	for (const suitName of variant.suits) {
+		if (['Black', 'Pink', 'Brown'].includes(suitName)) {
+			abbreviations.push(['k', 'i', 'n'][['Black', 'Pink', 'Brown'].indexOf(suitName)]);
+		} else {
+			const abbreviation = colors.find(color => color.name === suitName)?.abbreviation ?? suitName.charAt(0);
+			if (abbreviations.includes(abbreviation.toLowerCase()))
+				abbreviations.push(suitName.toLowerCase().split('').find(char => !abbreviations.includes(char)));
+			else
+				abbreviations.push(abbreviation.toLowerCase());
+
+		}
+	}
+	shortForms = abbreviations;
+}
 
 /**
  * Returns whether the card would be touched by the clue.
  * @param {Identity} card
- * @param {string[]} suits
+ * @param {Variant} variant
  * @param {Omit<Clue, 'target'>} clue
  */
-export function cardTouched(card, suits, clue) {
+export function cardTouched(card, variant, clue) {
 	const { type, value } = clue;
 	const { suitIndex, rank } = card;
-	const suit = suits[suitIndex];
+	const suit = variant.suits[suitIndex];
 
-	if (suit === 'Null')
+	if (suit === 'Null' || suit === 'Dark Null')
```

Comment:
I think this and the Omni check don't need to exist anymore? Since colour clues will hit whitish/rainbowish and rank clues will hit brownish/pinkish?